backends/dpdk/dpdkHelpers.cpp

```diff
@@ -688,7 +688,8 @@ bool ConvertStatementToDpdk::preorder(const IR::MethodCallStatement *s) {
                         incr = args->at(1)->expression;
                     if (!incr && value > 0) {
                         ::error(ErrorType::ERR_UNEXPECTED,
-                                "Expected packet length argument for %1%", a->object->getName());
+                                "Expected packet length argument for %1% of indirect counter",
+                                a->method->getName());
```

Comment:
description says after fix error should be like "count of indirect counter %1%", varname, but it doesn't match.